roles/pulp_common/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
 pulp_cache_dir: '/var/lib/pulp/tmp'
+pulp_artifact_dir: '/var/lib/pulp/artifact'
```

Comment:
If this is a public variable rather than a collection private one (no underscores at the beginning), then it needs to be documented in the role's README.md.

Reply:
Actually, this variable cannot be public. If the user changes that value, it will either break pulp or lead to a necessity to create migrations in the very installation.

Reply:
Yeah, prefix it with 2 underscores then: https://github.com/oasis-roles/meta_standards#naming-things